mobile/lib/ui/viewer/file/detail_page.dart

```diff
@@ -123,43 +132,48 @@ class _DetailPageState extends State<DetailPage> {
           _files!.length.toString() +
           " files .",
     );
-    return Scaffold(
-      appBar: PreferredSize(
-        preferredSize: const Size.fromHeight(80),
-        child: ValueListenableBuilder(
-          builder: (BuildContext context, int selectedIndex, _) {
-            return FileAppBar(
-              _files![selectedIndex],
-              _onFileRemoved,
-              100,
-              widget.config.mode == DetailPageMode.full,
-              enableFullScreenNotifier: _enableFullScreenNotifier,
-            );
-          },
-          valueListenable: _selectedIndexNotifier,
+    return WillPopScope.new(
```

Comment:
`WillPopScope` is deprecated, should migrate to using `PopScope`
https://docs.flutter.dev/release/breaking-changes/android-predictive-back